README.md

```diff
@@ -1,36 +1,44 @@
 
+### Part II: Bayesian Statistics
 * [Third chapter](https://datasciencejuliahackers.com:wq/03_probability_intro.jl.html): An introduction to probability, probability distributions and Bayes' interpretation.
 
-* [Fourth chapter](https://datasciencejuliahackers.com/04_naive_bayes.jl.html): using a Naive-Bayes approach we construct a simple spam email filter.
+* [Fourth chapter](https://datasciencejuliahackers.com/04_naive_bayes.jl.html): Using a Naive-Bayes approach we construct a simple spam email filter.
 
 * [Fifth chapter](https://datasciencejuliahackers.com/05_prob_prog_intro.jl.html): An introduction to Probabilistic Programming and some simple examples using the Turing.jl package.
 
-* [Sixth chapter](https://datasciencejuliahackers.com/06_gravity.jl.html): we estimate the gravity of Mars to compute the escape velocity, throwing stones and taking very simple measurements from it.
+* [Sixth chapter](https://datasciencejuliahackers.com/06_gravity.jl.html): We estimate the gravity of Mars to compute the escape velocity, throwing stones and taking very simple measurements from it.
+
+* [Seventh chapter](https://datasciencejuliahackers.com/07_football_simulation.jl.html): We use a hierarchical bayesian model to estimate latent variables that describe Premier League´s football teams.
 
-* [Seventh chapter](https://datasciencejuliahackers.com/07_football_simulation.jl.html): we use a hierarchical bayesian model to estimate latent variables that describe Premier League´s football teams.
+* [Eighth chapter](https://datasciencejuliahackers.com/08_basketball_shots.jl.html): We analyze how the scoring probability is affected by some variables, such as the distance from the hoop and the angle of shooting.
 
-* [Eighth chapter](https://datasciencejuliahackers.com/08_basketball_shots.jl.html): we analyze how the scoring probability is affected by some variables, such as the distance from the hoop and the angle of shooting.
+* [Ninth chapter](https://datasciencejuliahackers.com/09_optimal_pricing.jl.html): We solve a problem of optimal pricing optimization using a bayesian point of view.
 
-* [Nineth chapter](https://datasciencejuliahackers.com/09_optimal_pricing.jl.html): we solve a problem of optimal pricing optimization using a bayesian point of view.
+### Part III: Machine Learning
+* Work in progress
 
-* [Tenth chapter](https://datasciencejuliahackers.com/10_bees_vs_wasps.jl.html): in this chapter we give an overview of Machine Learning and we implement a simple convolutional neural network to solve a bees vs. wasps classification.
+### Part IV: Deep Learning
+* [Tenth chapter](https://datasciencejuliahackers.com/10_bees_vs_wasps.jl.html): Overview of Machine Learning and implementation of a simple convolutional neural network that is able to discriminate between pictures of bees and wasps..
```

Comment:
Please pay attention to details. After the copy paste there are two full stops at the end of this sentence. Review the whole text before submitting a PR